de.nagstamon.nagstamon.yml

```diff
@@ -0,0 +1,116 @@
+app-id: de.nagstamon.nagstamon
+runtime: org.kde.Platform
+runtime-version: '6.6'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.6'
+command: nagstamon
+rename-appdata-file: nagstamon.appdata.xml
+rename-desktop-file: nagstamon.desktop
+rename-icon: nagstamon
+build-options:
+  env:
+    - BASEAPP_REMOVE_WEBENGINE=1
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --persist=.nagstamon
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --env=QTWEBENGINEPROCESS_PATH=/app/bin/QtWebEngineProcess
+  - --filesystem=xdg-run/keyring
+  - --filesystem=home
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+modules:
+  - name: libsecret
```

Comment:
This should be from shared modules https://github.com/flathub/shared-modules/

Reply:
Changed in latest commit.